.gitlab-ci.yml

```diff
@@ -1,11 +1,16 @@
-netlify:
-  type: deploy
+image: python:3-slim-stretch
+stages:
+  - deploy
+
+build and push:
+  stage: deploy
+  before_script:
+     - apt-get update -qq && apt-get install -y -qq wget curl
   script:
-    # Stage
-    - curl -X POST -d {} ${WEBHOOK_STAGE}
-    # Prod
-    - curl -X POST -d {} ${WEBHOOK_PROD}
-    # Dead Man's Snitch Ping
-    - curl ${DMS}
-  only:
-    - schedules
+     - pip install awscli
+     - pip install --require-hashes --no-cache-dir -r requirements.txt    
+     - ./bin/mirror.sh
+     - aws s3 sync ./_site ${STAGE_BUCKET_PATH} --acl public-read --delete
+     - aws s3 sync ./_site ${PROD_BUCKET_PATH} --acl public-read --delete
```

Comment:
Why have stage if they're going to be exactly the same?

Reply:
copied from [here](https://github.com/mozmeao/lumbergh/blob/master/.gitlab-ci.yml#L5). I'm not sure. any ideas @glogiotatidis ?

Reply:
oh. that's because netlify was building those two deployments from different branches.

Reply:
while that job was set to run only on a schedule and update those deployments from those branches. To do the same thing here we'll need separate jobs for stage and prod with `only: master` and `only: prod` respectively.